test/videojs-http-streaming.test.js

```diff
@@ -1219,12 +1219,12 @@ QUnit.test('selects the correct rendition by tech dimensions', function(assert)
 
   assert.deepEqual(
     playlist.attributes.RESOLUTION,
-    {width: 960, height: 540},
+    {width: 396, height: 224},
```

Comment:
Although in many cases, particularly when there's a good number of available renditions, selecting the closest rendition rather than the closest plus one rendition makes sense, in a case like this one, the quality will be pretty degraded compared to the previous logic. We may want to make a reasonable allotment for how many fewer pixels we'll allow before just using the higher rendition, especially if there's plenty of bandwidth available to accommodate.